test/development/acceptance/error-recovery.test.ts

```diff
@@ -411,12 +411,31 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox %s', () => {
       `
     )
 
-    await new Promise((resolve) => setTimeout(resolve, 1000))
     expect(await session.hasRedbox()).toBe(true)
-    if (process.platform === 'win32') {
-      expect(await session.getRedboxSource()).toMatchSnapshot()
+    if (isTurbopack) {
+      expect(await session.getRedboxSource()).toMatchInlineSnapshot(`
+        "index.js (5:8) @ Error
+
+          3 | setInterval(() => {
+          4 | i++
+        > 5 | throw Error('no ' + i)
+            |      ^
+          6 | }, 1000)
+          7 | export default function FunctionNamed() {
+          8 | return <div />"
+    `)
     } else {
-      expect(await session.getRedboxSource()).toMatchSnapshot()
+      expect(await session.getRedboxSource()).toMatchInlineSnapshot(`
+        "index.js (5:8) @ Error
```

Comment:
this snapshot also looks same, is this intentional to have 2?

Reply:
Yeah it's to avoid having to update the tests yet again once we improve errors / the overlay further for Turbopack in the future. Also used it as an indicator of what wasn't handled yet.